client-next/src/components/MapView/MapView.tsx

```diff
@@ -35,6 +35,20 @@ export function MapView({
   const onMapDoubleClick = useMapDoubleClick({ tripQueryVariables, setTripQueryVariables });
   const [showContextPopup, setShowContextPopup] = useState<LngLat | null>(null);
   const [showPropsPopup, setShowPropsPopup] = useState<PopupData | null>(null);
+  const showFeaturePropPopup = (
+    e: mapboxgl.MapMouseEvent & {
+      features?: mapboxgl.MapboxGeoJSONFeature[] | undefined;
+    },
+    setShowPropsPopup: (value: ((prevState: PopupData | null) => PopupData | null) | PopupData | null) => void,
+  ) => {
+    if (e.features) {
+      // if you click on a cluster of map features it's possible that there are multiple
+      // to select from. we are using the first one instead of presenting a selection UI.
+      // you can always zoom in closer if you want to make a more specific click.
```

Comment:
That's neat. A potential improvement could be to instead zoom in one level if you encounter a cluster? Not critical for me.

Reply:
I'm not sure if that would be a good or bad UX but I'm willing to try it.

Reply:
It's something I've seen in other map implementations at least.

Reply:
We agreed that we will implement in an upcoming PR.